libbeat/logp/core.go

```diff
@@ -67,8 +67,9 @@ func Configure(cfg Config) error {
 	return ConfigureWithOutputs(cfg)
 }
 
-// Configure configures the logp package also sets up the following outputs to also be used
-// by the logger.
+// Configure configures the logger from passed in config and also adds the passed outputs to the logger.
+//
+// When passing in outputs the logger will log to each output.
```

Comment:
ConfigureWithOutputs is currently used by agent only. We have a dependency between this function and what the agent expects. Let's add a line saying:

```
// XXX: ConfigureWithOutputs is used by elastic-agent only (See file: ....). The agent requires that the output specified in the config object is confitured and merged with the logging outputs given. 
```

Reply:
Updated comment to match suggestion.